### PR TITLE
CRM-18157 - Make the mail format required

### PR DESCRIPTION
**Pull request by samuelsov**
cf. https://issues.civicrm.org/jira/browse/CRM-18157

### CODE REVIEW

**Comment by seamuslee001** [comment]
Hi @samuelsov Samuel can you please fix the style error found by Jenkins https://test.civicrm.org/job/CiviCRM-Core-PR/11747/checkstyleResult/new/ (making true TRUE)

Cheers

Seamus

**Comment by samuelsov** [comment]
It should be good... i should definitively have some way to check for checkstyle before submitting the PR

**Comment by seamuslee001** [comment]
@samuelsov If you download civicrm-buildkit and install it, there is a tool civilint which does the style checking.

**Comment by samuelsov** [comment]
The only failing tests seems to be "incomplete tests" (unrelated) and it's really a minor patch. I think there is no much risks in merging this one.

File: CRM/Contact/Form/Edit/CommunicationPreferences.php [hunk 1]
```diff
@@ -84,7 +84,7 @@ public static function buildQuickForm(&$form) {
     //using for display purpose.
     $form->assign('commPreference', $commPreference);
 
-    $form->addField('preferred_mail_format', array('entity' => 'contact', 'label' => ts('Email Format')));
+    $form->addField('preferred_mail_format', array('entity' => 'contact', 'label' => ts('Email Format')), TRUE);
```
Comment (jitendrapurohit) [inline CRM/Contact/Form/Edit/CommunicationPreferences.php]: This throws a validation error when attempting to create a new contact from "New Individual" block.

**Pull request merged**